src/main/java/uk/nhs/hee/tis/trainee/forms/config/InterceptorConfiguration.java

```diff
@@ -35,9 +36,19 @@
 @Configuration
 public class InterceptorConfiguration implements WebMvcConfigurer {
 
+  // Endpoints are a mix of authenticated (public) and unauthenticated (internal), limit
+  // trainee ID verification to LTFT, COJ and FormR endpoints for now.
+  protected static final String[] TRAINEE_ID_APIS
+      = {"/api/coj",
+        "/api/formr-parta", "/api/formr-partas", "/api/formr-parta/**",
+        "/api/formr-partb", "/api/formr-partbs", "/api/formr-partb/**",
+        "/api/ltft", "/api/ltft/**"};
```

Comment:
These are a bit uglier than I was hoping for, but I guess we don't have the same flexibility.